Lib/multiprocessing/queues.py

```diff
@@ -20,8 +20,6 @@
 
 from queue import Empty, Full
 
-import _multiprocessing
```

Comment:
How sure are you that this import doesn't have a sneaky side effect that's relied upon here? After all it's the C accelerator, things would presumably keep working without it, just slower, so passing tests might not be enough signal.

Reply:
Yes, I was unsure about this one. I'll just revert it for now.